code/Test_definitions/quality-on-demand-createSession.feature

```diff
@@ -275,19 +275,6 @@ Feature: CAMARA Quality On Demand API, v0.11.1 - Operation createSession
 
     # Errors 403
```

Comment:
do we need to keep the line if there is not test case in this section?

Reply:
I added 403 "missing scope" errors for all feature files